zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -829,10 +832,24 @@ impl<F: Field> ExecutionConfig<F> {
             }
         }
         for column in cell_manager.columns().iter() {
-            if let CellType::LookupByte = column.cell_type {
-                meta.lookup_any("Byte lookup", |meta| {
-                    let byte_table_expression = byte_table.table_exprs(meta)[0].clone();
-                    vec![(column.expr(), byte_table_expression)]
+            if let CellType::LookupU8 = column.cell_type {
+                meta.lookup_any("u8 lookup", |meta| {
+                    vec![column.expr()]
+                        .into_iter()
+                        .zip(u8_table.table_exprs(meta).into_iter())
+                        .map(|(expr, table)| (expr, table))
+                        .collect()
+                });
+            }
+        }
+        for column in cell_manager.columns().iter() {
```

Comment:
Nitpick: Can we merge this loop with the previous one (in line 834)?

Reply:
Sure and good idea! Rewrite this part `match` syntax https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1427/commits/13c03a9343241f2847884990713d4787604388d1